src/rust/storage/seg/src/ttl_buckets/ttl_buckets.rs

```diff
@@ -0,0 +1,112 @@
+// Copyright 2021 Twitter, Inc.
+// Licensed under the Apache License, Version 2.0
+// http://www.apache.org/licenses/LICENSE-2.0
+
+//! A collection of [`TtlBucket`]s which covers the full range of TTLs.
+
+use crate::*;
+
+const N_BUCKET_PER_STEP_N_BIT: usize = 8;
```

Comment:
I feel this file could use more comments, or a link to the paper with relevant section pointed out.

Reply:
See if it's better now or if you feel it needs more still.

Reply:
Great!